morse_decode.rb

```diff
@@ -0,0 +1,53 @@
+# Define the dictionnary of correspondance
+@dictionary = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+# get the letter from the dictionary
+def get_letter(letter)
+  @current_letter = @dictionary[letter]
+  @current_letter
+end
+# Decode a word
+
+def decode_word(word)
+  @letters = word.split
+  @current_word = ''
+  @letters.each do |letter|
+    get_letter(letter)
+    @current_word += get_letter(letter)
+  end
+  print "#{@current_word} "
+end
+
+# decode a compelete message
+def decode_message(message)
+  @words = message.split('   ')
+  @words.each do |word|
```

Comment:
- Good job done so far, but your output is not displaying correctly, please fix this 👍 👍, your output should only display te decoded message.
<img width="717" alt="vuhhbh" src="https://user-images.githubusercontent.com/73607512/216592624-71a6231c-a220-46d7-a47e-6283fcb87251.PNG">

Reply:
Hi 
Thank you for your feedback. When I run the code locally it gives me only the decoded message without the code